fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -6215,7 +6215,8 @@ private ChangedTransactionDetail handleRefundTransaction(final LoanTransaction l
     }
 
     public void handleChargebackTransaction(final LoanTransaction chargebackTransaction,
-            final LoanLifecycleStateMachine loanLifecycleStateMachine) {
+            final LoanLifecycleStateMachine loanLifecycleStateMachine, final List<Long> existingTransactionIds,
```

Comment:
I don't really like this change because it'll bring side-effects to an already complex codebase.

Why can't we just do the findExistingTransactionIds and findExistingReversedTransactionIds where this is being called?

Reply:
To be honest It's the same, I mean, It's the unique part of the code that uses this.

Reply:
Done, updated